cegs_portal/search/templates/search/v1/dna_feature.html

```diff
@@ -259,29 +254,29 @@ <h1>
                 <div class="title-separator"></div>
                 <table class="data-table">
                     <tr>
-                        <th class="chrom-first-end-cap chrom-light-band">Accession ID</th>
-                        <th class="chrom-dark-band">Type</th>
-                        <th class="chrom-light-band">Cell Line</th>
-                        <th class="chrom-light-band chrom-right-centromere">Location</th>
-                        <th class="chrom-light-band chrom-left-centromere">Strand</th>
-                        <th class="chrom-last-end-cap chrom-dark-band">Reference Genome</th>
+                        <th class="chrom-first-end-cap chrom-dark-band">Type</th>
+                        <th class="chrom-light-band chrom-right-centromere  ">Cell Line</th>
+                        <th class="chrom-left-centromere chrom-light-band">Location</th>
+                        <th class="chrom-last-end-cap chrom-light-band">Feature Details</th>
```

Comment:
I think just `Details`? We know these are DNA Features from the table container header. That also matches the place we already do this.